Assets/Scripts/Control&Input/PlayerMovement.cs

```diff
@@ -186,18 +169,15 @@ private void UpdatePosition(Vector3 input)
                     // Strafe slightly with less drag.
                     body.drag = airDrag;
                     body.AddForce(input * strafeForceInAir, ForceMode.VelocityChange);
-                    body.AddForce(Vector3.down * MarsGravity, ForceMode.Acceleration);
+                    body.AddForce(Vector3.down * marsGravity * 3.5f, ForceMode.Acceleration);
```

Comment:
Why adjust gravity here and not in the constant? And why adjust it in the first place? We changed it to match Mars's gravity (though I do agree that we could turn it up a few notches)

Reply:
removed it. It's back to how it was